### PR TITLE
AzDO handles submodules poorly, force an update on all submodules.

### PR DESCRIPTION
**Pull request by jeminor**

### CODE REVIEW

File: scripts/azdo/pipeline.yml [hunk 1]
```diff
@@ -20,5 +20,6 @@ jobs:
   - checkout: self
     submodules: true
   - script: |
+      git submodule update --remote --checkout
```
Comment (shruthi-ravi) [inline scripts/azdo/pipeline.yml:23]: Do you need to include the --init option?

**Review by shruthi-ravi — APPROVED** [review]
Looks good.

File: scripts/azdo/pipeline.yml (hunk 1 above)
Comment (jeminor) [reply scripts/azdo/pipeline.yml:23]: That's a good question, but no (the submodules: true does that much at least).

File: scripts/azdo/pipeline.yml [hunk 2]
```diff
@@ -20,5 +20,6 @@ jobs:
...
       source ni-oe-init-build-env
```
Comment (amstewart) [inline scripts/azdo/pipeline.yml:24]: I've pulled the PR which updates this init script with the correct (new) environment variables. Change this source line to [this file](https://github.com/ni/nilrt/blob/nilrt/master/dunfell/nilrt-build-init.env) to populate `NILRT_ROOT`.

**Review by amstewart — CHANGES_REQUESTED** [review]

**Comment by amstewart** [comment]
Is this PR still live, @jeminor ?

**Pull request closed without merging**